src/Sound/Tidal/UI.hs

```diff
@@ -756,7 +758,7 @@ euclid = tParam2 _euclid
 
 _euclid :: Int -> Int -> Pattern a -> Pattern a
 _euclid n k a | n >= 0 = fastcat $ fmap (bool silence a) $ bjorklund (n,k)
-              | otherwise = _euclidInv (-n) k a
+              | otherwise = fastcat $ fmap (bool a silence) $ bjorklund (-n,k)
```

Comment:
Is this correct? `bjorklund` should be called with `(n,k)`

Reply:
I think it is correct, this is the case of `n < 0`, so `-n > 0` (otherwise bjorklund wouldn't produce anything). Note the reversed order of `a` and `silence`

Reply:
Yes, right, did a wrong assumption! So ok for me 👍